windows/CodePush.Shared/CodePushConstants.cs

```diff
@@ -27,5 +25,11 @@ internal class CodePushConstants
         internal const string RelativeBundlePathKey = "bundlePath";
         internal const string StatusFileName = "codepush.json";
         internal const string UnzippedFolderName = "unzipped";
+#if WINDOWS_UWP
+        internal const string AssetsBundlePrefix = "ms-appx:///ReactAssets/";
+        internal const string FileBundlePrefix = "ms-appdata:///local";
+#else
```

Comment:
Since fix in [PR#950](https://github.com/ReactWindows/react-native-windows/pull/950) **ReactNative.Net46** uses absolute path, without prefix. Therefore all prefixes were removed for Net46 solution.